<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedControl</class>
 <widget class="QDialog" name="LedControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_led1_ctl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>LED1控制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_led2_ctl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>LED2控制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_led3_ctl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>LED3控制</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_led1_open">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led1Group</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_led1_close">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led1Group</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_led2_open">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led2Group</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_led2_close">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led2Group</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_led3_open">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led3Group</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_led3_close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Led3Group</string>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>20</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>20</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_leds_open">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">LedsGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_leds_close">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">LedsGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="button_leds_ctl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>总控制台</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>16</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>16</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Return">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>状态:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Led1Group"/>
  <buttongroup name="Led2Group"/>
  <buttongroup name="Led3Group"/>
  <buttongroup name="LedsGroup"/>
 </buttongroups>
</ui>
